Dockerfile

```diff
@@ -14,17 +15,17 @@ RUN apt-get install python2.7 python2.7-dev python \
                     python-pysam \
                     python-scipy \
                     -y
-RUN apt-get install software-properties-common python-software-properties -y
+RUN apt-get install software-properties-common -y
 RUN apt-get clean -y
 
 RUN pip install bx-python
 
-RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
-    add-apt-repository -y ppa:webupd8team/java && \
+RUN echo oracle-java11-installer shared/accepted-oracle-license-v1-2 select true | debconf-set-selections && \
```

Comment:
Internally we've switched to openjdk8 without problems. Also solves the license issues with Oracle Java which would still exist with this fix.